### ISSUE TITLE
Not working global search in Russian

### ISSUE THREAD

**Issue by AircraftRu** [glpi-project/glpi#1386]
Not working global search in Russian, english search works.
GLPI ver. 9.1.1

And search does not work on the description of the ticket

**Comment by AnaelMobilia** [comment]
Hello,
I just made a test and found my ticket with the global search using russian words.
The search was only occuring on the title of the ticket.

**Comment by trasher** [comment]
Sounds like it has been fixed in 9.1/bugfixes branch.

Google translate give me "Значит ли это работает?" as a translation for "Does that works?"; I've used this in a ticket title an make a global search with "Значит", I can see the relevant ticket.
Search query WHERE part is:
```
`glpi_tickets`.`is_deleted` = '0' AND ( `glpi_tickets`.`status` IN ('1','2','3','4') AND ( `glpi_tickets`.`id` LIKE '%Значит%' OR `glpi_tickets`.`name` LIKE '%Значит%' OR `glpi_entities`.`completename` LIKE '%Значит%' OR `glpi_tickets`.`status` LIKE '%Значит%' OR CONVERT(`glpi_tickets`.`date_mod` USING utf8) LIKE '%Значит%' ) )
```

And no, this does not search in ticket description, if you think it should, please open another specific ticket to ask for this enhancement.
